packages/hierarchies-react/src/presentation-hierarchies-react/internal/TreeModel.ts

```diff
@@ -26,59 +27,52 @@ export interface TreeModelHierarchyNode {
   isSelected?: boolean;
   hierarchyLimit?: number | "unbounded";
   instanceFilter?: GenericInstanceFilter;
+  error?: TreeModelError;
 }
 
 /** @internal */
-export interface TreeModelGenericInfoNode {
+export interface TreeModelGenericError {
   id: string;
   parentId: string | undefined;
```

Comment:
1. do these error types need to have `parentId`, now that they're assigned to the parent node?
2. is there a reason to have very similar error types, e.g. `TreeModelGenericError` vs `PresentationGenericError`?

Reply:
had some issues merging with the newest `next` version seems the removal of `parentId` has been reverted will remove it. For the differences of `Model` and `Hierarchy` errors i think if we were to remove the model part entirely we could merge them. But having nodes as `Model` nodes and `errors` as hierarchy errors could be weird as we then expose hierarchy parts to the loader.

Reply:
I don't think we need `TreeModel*Error` types anymore. It made sense when they were part of tree model itself but as they now are part of node you can use `Presentation*Error` as a type for error property on both `TreeModelNode` and `PresetnationNode`

Reply:
Removed the type, model and `PresentationHierarchyNode` both now use  `ErrorInfo`